src/main/java/gregtech/api/items/metaitem/stats/IItemHUDProvider.java

```diff
@@ -0,0 +1,38 @@
+package gregtech.api.items.metaitem.stats;
+
+import net.minecraft.item.ItemStack;
+import net.minecraftforge.fml.relauncher.Side;
+import net.minecraftforge.fml.relauncher.SideOnly;
+
+import javax.annotation.Nonnull;
+
+/**
+ * Provides a drawable HUD for the item
+ */
+public interface IItemHUDProvider extends IItemComponent {
```

Comment:
It might be better to make `ArmorLogicSuite` implement this interface instead of the various different base armors